bot.py

```diff
@@ -95,6 +95,16 @@ async def on_ready(self):
 class Setup(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        
+    # Helper to send a message to the author of the command about what they did wrong
+    async def sendErrorToAuthor(self, ctx, error=None):
```

Comment:
Instead of duping this, move it into the botcBot class so we can call it from either Cog via `self.bot.sendErrorToAuthor`